supervisor/store/data.py

```diff
@@ -87,7 +88,9 @@ def _read_addons_folder(self, path: Path, repository: Dict) -> None:
                 IssueType.CORRUPT_REPOSITORY,
                 ContextType.STORE,
                 reference=path.stem,
-                suggestions=[SuggestionType.EXECUTE_RESET],
+                suggestions=[SuggestionType.EXECUTE_RESET]
```

Comment:
Move this if/else above `self.sys_resolution.create_issue(` to make it more readable